sdk/go/common/apitype/ai.go

```diff
@@ -16,14 +16,46 @@ package apitype
 
 import "encoding/json"
 
-// Requests
+// SummarizeUpdateRequest
 
 type CopilotSummarizeUpdateRequest struct {
 	Query           string                 `json:"query"`
 	State           CopilotState           `json:"state"`
-	DirectSkillCall CopilotDirectSkillCall `json:"directSkillCall"`
+	DirectSkillCall CopilotSummarizeUpdate `json:"directSkillCall"`
 }
 
+type CopilotSummarizeUpdate struct {
+	Skill  string                       `json:"skill"` // The skill to call. e.g. "summarizeUpdate"
+	Params CopilotSummarizeUpdateParams `json:"params"`
+}
+
+type CopilotSummarizeUpdateParams struct {
+	PulumiUpdateOutput string `json:"pulumiUpdateOutput"` // The Pulumi update output to summarize.
```

Comment:
Instead of duplicating all these types, wouldn't a better API be returning just `pulumiOutput`?

Reply:
Agree, I think I prefer that for aesthetic consistency across the different "skill calls" that relate to processing pulumi output.

I would avoid sharing the "skill call param" types between different skills though, just to avoid mix ups etc. 

Added some type embedding stuff to avoid the top level field duplication between the diff requests.